src/styles/blocks/card.scss

```
@@ -0,0 +1,94 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-top: 40px;
+    color: $main-accent;
+    font-size: $font-size-m;
+    font-weight: $medium;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+    color: $secondary;
+    font-size: $font-size-s;
+    font-weight: $regular;
+    line-height: 14px;
+  }
+
+  &__review {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__review-quantity {
+    margin: 0;
+    color: $main-accent;
+    text-align: right;
+    font-size: $font-size-s;
+    font-weight: $regular;
+    line-height: 14px;
+  }
```

try to use scss syntax 
```suggestion
  &__review {
    margin-top: 16px;
    display: flex;
    justify-content: space-between;
    align-items: center;
    
    &-quantity {
    margin: 0;
    color: $main-accent;
    text-align: right;
    font-size: $font-size-s;
    font-weight: $regular;
    line-height: 14px;
   }
  }
```